static/app/components/lazyRender.tsx

```diff
@@ -61,6 +61,10 @@ export function LazyRender(props: LazyRenderProps) {
         for (const entry of entries) {
           if (entry.isIntersecting) {
             setVisible(true);
+            if (node) {
```

Comment:
This was a bug in the previous implementation - we need to clear the placeholder height so that we dont end up limiting the height of the rendered content